Submitted By: Tushar Teredesai <tushar@linuxfromscratch.org>
Date: 2004-01-24
Initial Package Version: 1.4.2
Origin: Gentoo
Description: Link to shared libstd++.

$LastChangedBy$
$Date$

--- j2sdk/j2se/make/common/Defs-linux.gmk.jj	2003-12-20 10:42:01.745988008 +0000
+++ j2sdk/j2se/make/common/Defs-linux.gmk	2003-12-20 10:42:21.892925208 +0000
@@ -23,7 +23,7 @@
 #   LINTFLAGS     (set $(OTHER_LINTFLAGS) instead)
 
 # statically link libstdc++ before C++ ABI is stablized on Linux
-STATIC_CXX     = true
+STATIC_CXX     = false
 
 # define these to avoid picking up ones from aliases or from
 # non-standard locations
